dashboard/src/locales/en.json

```diff
@@ -1 +1,62 @@
-{}
+{
```

Comment:
can we gitignore this file? (to avoid confusion with dashboard/lang/en.json)

Reply:
Mmm, I don't really know if it's a good option. If we ignore files under `src/locales/*.json` users cloning the repo will get many console errors (despite the strings will fallback to the default ones) because the compiled AST translation is not present.
If we ignore them, I think we should modify the `start` script from the `package.json` in order to add the language compilation beforehand.

Reply:
yes, I would do so (modify the `start ` method). The same approach we do with the CSS files (compiled from scss source). You may need to update the `build` script as well because that's the one executed in the CI:

```
    "start": "npm-run-all -p watch-css start-js",
    "build": "npm-run-all build-css build-js",
```

Reply:
Great, good idea. Do you want me to create a separate PR for that? Or just adding these changes here is fine?

Reply:
whatever is easier for you

Reply:
Perhaps, to keep this pr as simple as possible, let's do it on another PR.
If it's OK with you, I'm gonna merge this one and will send the other PR asap.

Reply:
yep